apps/web/src/components/providers/AuthProvider.tsx

```diff
@@ -26,6 +27,19 @@ export const useAuthContext = (): UserContext => useContext(AuthContext);
 
 export const AuthProvider = ({ children }: { children: React.ReactNode }) => {
   const { token, setToken, user, organization, logout, jwtPayload, organizations } = useAuthController();
+  const ldClient = useLDClient();
+
+  useEffect(() => {
+    if (!organization?._id) {
+      return;
+    }
+
+    ldClient?.identify({
+      kind: 'Organization',
+      key: organization._id,
+      name: organization.name,
+    });
+  }, [organization?._id, ldClient, organization?.name]);
```

Comment:
What do you think about extracting this into a separate hook? That way, if we change FF providers we can keep the code change isolated.

```typescript
import { useLDClient } from 'launchdarkly-react-client-sdk';

const useFeatureFlagClient = (organization: IOrganizationEntity) => {
  const ldClient = useLDClient();

  useEffect(() => {
    if (!organization?._id) {
      return;
    }

    ldClient?.identify({
      kind: 'Organization',
      key: organization._id,
      name: organization.name,
    });
  }, [organization?._id, ldClient, organization?.name]);
}
```

Reply:
We many need to change in the future due to the change with the platform so I think this would be a good idea.